### DISCUSSION TITLE
OpenAI proxy server like api.pawan.krd

### DISCUSSION THREAD

**Discussion by ThierryBachmann** [RageAgainstThePixel/com.openai.unity#180] [Q&A] [answered]
Request ChatGPT thru a reverse proxy like https://discord.pawan.krd/

Pros: free API key
Cons: slower than original

But perhaps, I didn't find how to with the current version?

**Comment by StephenHodgson** [comment, marked as answer]
Here is already support for any proxy you desire to use:

https://github.com/RageAgainstThePixel/com.openai.unity#openai-api-proxy
